main/integration-tests/support/resources/users/users.js

```diff
@@ -54,12 +54,21 @@ class Users extends CollectionResource {
     };
   }
 
+  async cleanup(user) {
+    const resource = new User({ clientSession: this.clientSession, id: user.uid, parent: this });
+    return resource.delete();
+  }
+
```

Comment:
Any reason why you have added this back?  I removed it in my previous PR.  The create method is putting a cleanup task and doing the right thing by delegating the deletion to the instance resource.  Not sure why you want this here, maybe I missed something?